lockfile.json

```diff
@@ -198,7 +198,7 @@
     "revision": "0e36bed171768908f331ff7dff9d956bae016efb"
   },
   "yang": {
-    "revision": "eb4ace1fdc4954d7e4f0687c687219f78a6ce64e"
+    "revision": "cc81c9a8f40e77567e66cfd83fd532817ca256b9"
```

Comment:
This version should not be changed.

Reply:
But that's the point of this PR to make the automated update mergeable

Reply:
No but there's two version changes here : one for `teal` and the other one for `yang` which should not be there

Reply:
The versions shouldn't be touched by humans. Just let the GitHub bot do it's work

Reply:
What I say is simple:
1. You said you are updating teal
2. The changes in the lockfile contain both a teal update and a yang update
3. That's not right.

Reply:
But that was our current workflow that we fix all the breakages of an updates in seperate commits for each language. That scales much better when two or more languages are broken.

I've edited the lockfile commit: it's only teal now.